sdcm/results_analyze/__init__.py

```diff
@@ -157,8 +157,21 @@ def check_regression(self, test_id, data, is_gce=False):  # pylint: disable=too-
         test_start_time = datetime.utcfromtimestamp(float(doc["_source"]["test_details"]["start_time"]))
         test_version_info = self._test_version(doc)
         test_version = test_version_info['version']
-        subject = f'Performance Regression Compare Results - {test_name} - {test_version} - {str(test_start_time)}'
+        last_events = dict()
+        events_summary = dict()
+        last_events['CRITICAL'] = doc["_source"]["events"]["CRITICAL"]
+        events_summary['CRITICAL'] = len(last_events['CRITICAL'])
+
+        last_events['ERROR'] = doc["_source"]["events"]["ERROR"]
+        events_summary['ERROR'] = len(last_events['ERROR'])
+
+        last_events['WARNING'] = doc["_source"]["events"]["WARNING"]
+        events_summary['WARNING'] = len(last_events['WARNING'])
+        subject = f'Performance Regression Compare Results (latency during mgmt ops) -' \
```

Comment:
I think, new method which will return ready dicts will be better. Also you can add some checks there if such fields by some reasons will be missed in document.

Reply:
this was done, and is being tested [here](https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/fabio/job/latency_automatic_calculation_read/17/)